### DISCUSSION TITLE
Zoomming Gridmap not show green and red square

### DISCUSSION THREAD

**Discussion by emics** [magicbug/Cloudlog#3130] [General]
With v2.6.14 in Gridmap i can see all grid worked and confirmed (red and green square).
If I zoom in a 4 chars square i can see detailed grid (6 chars) but i don't have the square coloured green and red although my QSO are correct with 6 chars grid.

have you the same ?
thanks

**Comment by magicbug** [comment]
I've a weird feeling thats always been the case but certainly something that needs resolved

**Comment by magicbug** [comment]
This now has an issue at https://github.com/magicbug/Cloudlog/issues/3136
